File: server/FlightLog-server/src/main/java/bitc/fullstack503/flightlog/flightlogserver/dto/dFlightDTO.java
package bitc.fullstack503.flightlog.flightlogserver.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
public class dFlightDTO {

  @JsonProperty("airlineKorean")
  private String airlineKorean;

  @JsonProperty("arrivalcity")
  private String arrivalcity;

  @JsonProperty("domesticArrivalTime")
  private String domesticArrivalTime;

  @JsonProperty("domesticEddate")
  private String domesticEddate;

  @JsonProperty("domesticMon")
  private String domesticMon;

  @JsonProperty("domesticTue")
  private String domesticTue;

  @JsonProperty("domesticWed")
  private String domesticWed;

  @JsonProperty("domesticThu")
  private String domesticThu;

  @JsonProperty("domesticFri")
  private String domesticFri;

  @JsonProperty("domesticSat")
  private String domesticSat;

  @JsonProperty("domesticSun")
  private String domesticSun;

  @JsonProperty("domesticNum")
  private String domesticNum;

  @JsonProperty("domesticStdate")
  private String domesticStdate;

  @JsonProperty("domesticStartTime")
  private String domesticStartTime;

  @JsonProperty("startcity")
  private String startcity;



}

File: app/app/src/main/java/bitc/fullstack/FlightLog/dto/dFlightDTO.kt
package bitc.fullstack.FlightLog.dto

import com.google.gson.annotations.SerializedName

// 서버로 전송하는 국내 항공 정보
data class dFlightDTO(
    @SerializedName("airlineKorean")
    var airlineKorean: String,

    @SerializedName("arrivalcity")
    var arrivalcity: String,

    @SerializedName("domesticArrivalTime")
    var domesticArrivalTime: String,

    @SerializedName("domesticEddate")
    var domesticEddate: String,

    @SerializedName("domesticMon")
    var domesticMon: String,

    @SerializedName("domesticTue")
    var domesticTue: String,

    @SerializedName("domesticWed")
    var domesticWed: String,

    @SerializedName("domesticThu")
    var domesticThu: String,

    @SerializedName("domesticFri")
    var domesticFri: String,

    @SerializedName("domesticSat")
    var domesticSat: String,

    @SerializedName("domesticSun")
    var domesticSun: String,

    @SerializedName("domesticNum")
    var domesticNum: String,

    @SerializedName("domesticStdate")
    var domesticStdate: String,

    @SerializedName("domesticStartTime")
    var domesticStartTime: String,

    @SerializedName("startcity")
    var startcity: String
)
